deepchem/models/torch_models/layers.py

```diff
@@ -0,0 +1,44 @@
+try:
+  import torch
+  import torch.nn as nn
+except:
+  raise ImportError('These classes require Torch to be installed.')
+
+
+class LayerNorm(nn.Module):
+  """Apply Layer Normalization to input.
+
+    The layer takes input and applies layer normalization to it.
+
+    References
+    ----------
+    .. [1] Lukasz Maziarka et al. "Molecule Attention Transformer" Graph Representation Learning workshop and Machine Learning and the Physical Sciences workshop at NeurIPS 2019. 2020. https://arxiv.org/abs/2002.08264
```

Comment:
Are you sure LayerNorm was discovered in this paper? Its a more general thing

Reply:
Thank you, I checked the equation again and other than slight differences it's the same. I have changed it to ScaleNorm now.